lib/i18n_screwdriver.rb

```diff
@@ -10,9 +10,13 @@ def self.filename_for_locale(locale)
     File.join("config", "locales", "application.#{locale}.yml")
   end
 
-  def self.generate_key(string)
-    string = string.strip
-    (string =~ /^:[a-z][a-z0-9_]*$/) ? string : Digest::MD5.hexdigest(string)
+  def self.generate_key(source)
```

Comment:
could you please use this syntax:


    def self.generate_key(source)
      return ":#{source}" if source.is_a?(Symbol)
      source = source.strip
      (source =~ /^:[a-z][a-z0-9_]*$/) ? source : Digest::MD5.hexdigest(source)
    end